src/ZeroEventHubClient/Client.cs

```diff
@@ -62,6 +62,26 @@ public Client(string url, int partitionCount, Func<HttpRequestMessage, Task> req
         await FetchEvents(cursors, pageSizeHint, eventReceiver, new List<string>(), cancellationToken);
     }
 
+    /// <summary>
+    /// Fetch events from the server
+    /// </summary>
+    /// <param name="cursor">A single cursor to be used in the request.</param>
+    /// <param name="pageSizeHint">An hint for the page size of the response.
+    /// Set to 0 if the server should decide the size returned</param>
+    /// <param name="eventReceiver">An event receiver to handle the received events.</param>
+    /// <exception cref="ArgumentException">If cursors are missing.</exception>
+    /// <exception cref="MalformedResponseException">If returned response body could not be parsed.</exception>
+    /// <exception cref="HttpRequestException">If response status code does not indicate success.</exception>
+    public async Task FetchEvents(
+        Cursor cursor,
+        int pageSizeHint,
+        IEventReceiver eventReceiver,
+        CancellationToken cancellationToken = default)
+    {
+        var cursors = new List<Cursor> { cursor };
+        await FetchEvents(cursors, pageSizeHint, eventReceiver, new List<string>(), cancellationToken);
```

Comment:
```suggestion
        await FetchEvents(new List<Cursor> { cursor }, pageSizeHint, eventReceiver, new List<string>(), cancellationToken);
```

Reply:
I think you could also write this as "new [] { cursor }" but it probably doesn't make a lot of difference.

Reply:
Good point, i will try the suggestion you suggested @larserikcowardeek  thanks!